mllib/src/test/scala/org/apache/spark/ml/feature/CountVectorizerSuite.scala

```diff
@@ -183,6 +183,26 @@ class CountVectorizerSuite extends SparkFunSuite with MLlibTestSparkContext
       case Row(features: Vector, expected: Vector) =>
         assert(features ~== expected absTol 1e-14)
     }
+
+    // CountVectorizer test
+    val df2 = sqlContext.createDataFrame(Seq(
```

Comment:
@BryanCutler I want to keep the "a" "b" "c" order after fit. I will remove the new df and reuse the same df. Will update soon.

Thanks!

Miao